R/export.R

```diff
@@ -13,7 +13,8 @@
 #' @param package_cache Cache downloaded binary WebAssembly packages. Defaults
 #'    to `TRUE`.
 #' @param assets_version The version of the Shinylive assets to use in the
-#'    exported app. Defaults to [assets_version()].
+#'    exported app. Defaults to [assets_version()]. Note, not all custom assets
+#'    versions may work with this release of \pkg{shinylive}.
```

Comment:
What kinds of issues do you have in mind? This is a decent "cover our butts" warning but maybe we could be more specific. Maybe we should also link to the shinylive release page so it's clear what our options are?

Reply:
Yes, more of a "cover our butts" situation. 

Yes, a link would be good!